.github/workflows/main_workflow.yml

```diff
@@ -149,7 +149,7 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest, macos-latest, windows-latest ]
-        python-version: [ "3.7", "3.8", "3.9", "3.10" ]
+        python-version: [ "3.8", "3.9", "3.10", "3.11" ]
```

Comment:
If we require Python >= 3.10 below, why do we have 3.8, 3.9 here?

Reply:
Yep, had to update README.md